src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1353,7 +1353,7 @@ void os::jvm_path(char *buf, jint buflen) {
         if (0 == access(buf, F_OK)) {
           // Use current module name "libjvm.so"
           len = strlen(buf);
-          snprintf(buf + len, buflen-len, "/hotspot/libjvm.so");
+          snprintf(buf + len, buflen-len, "/server/libjvm.so");
```

Comment:
Still no idea what this is supposed to do but there's a comment above at 1317 that should be fixed too.